src/playground/with-layout/chat/chat-custom-message.component.html

```diff
@@ -0,0 +1,34 @@
+<nb-chat title="Nebular Conversational UI" size="large">
+    <nb-chat-message *ngFor="let msg of messages"
+        [type]="msg.type"
+        [message]="msg.text"
+        [reply]="msg.reply"
+        [sender]="msg.user.name"
+        [date]="msg.date"
+        [files]="msg.files"
+        [quote]="msg.quote"
+        [latitude]="msg.latitude"
+        [longitude]="msg.longitude"
+        [avatar]="msg.user.avatar"
+        [customMessageData]="msg.customMessageData">
```

Comment:
What do you think about decreasing message types to text and two custom ones, so we could remove most of the `[input]="value"` expressions here to simplify example, so user could focus on custom message examples?

Reply:
Nice idea. Deal!